### PR TITLE
Cleanup unexpected success for Swift

### PR DESCRIPTION
**Pull request by JDevlieghere**
I'm going through the unexpected successes in the test suite. I'm re-enalbing the ones for which the radar suggests they might/are fixed on the condition that they works on all of my machines. Probably some of them will still fail on the bots, which we'll hopefully find out here.

### CODE REVIEW

**Review by dcci — APPROVED** [review]
LGTM. Thanks for your help with this.
I understand it might sound as thankless but it will go a long way.

**Comment by dcci** [comment]
ehm, let me rephrase that in English. I understand this may be a thankless job, but it's very important/appreciated.

**Comment by JDevlieghere** [comment]
@dcci Not at all, I'm glad to help improve our test suite!

**Comment by JDevlieghere** [comment]
@swift-ci please test

**Comment by dcci** [comment]
@JDevlieghere also, feel free to speculatively run tests on the bots while you wait of the reviews, as it may save a little of time :)

**Comment by JDevlieghere** [comment]
Yeah, I forgot this time. Thanks for the reminder!

**Comment by JDevlieghere** [comment]
@swift-ci please test
